partiql-tests-data/partiql-tests-schema.isl

```diff
@@ -107,7 +122,8 @@ type::{
     type: struct,
     fields: {
         result: { type: symbol, valid_values: [EvaluationSuccess], occurs: required },
-        output: { occurs: required }
+        output: { occurs: required },
```

Comment:
I considered modeling output as the `result` for `EvaluationSuccess` cases but this made the schema validation a lot more complicated. The `result` field has been a symbol and allowing it to take `ouput`'s value (i.e. any Ion value) can result in a duplicate matching type with `EvaluationFail`. So I chose to keep the `output` field separate.